lib/maths/CTimeSeriesModel.cc

```diff
@@ -148,12 +127,12 @@ const std::string VERSION_6_5_TAG("6.5");
 const std::string ID_6_3_TAG{"a"};
 const std::string IS_NON_NEGATIVE_6_3_TAG{"b"};
 const std::string IS_FORECASTABLE_6_3_TAG{"c"};
-const std::string RNG_6_3_TAG{"d"};
+//const std::string RNG_6_3_TAG{"d"};
```

Comment:
Do we usually just comment out removed tags? Might be better to include a message that includes the version of removal.